app/assets/stylesheets/application.scss

```diff
@@ -42,6 +42,21 @@ $govuk-global-styles: true;
     label {
       @extend .govuk-\!-font-weight-bold;
     }
+    div.hint {
```

Comment:
Do we need the `div` selectors here, and can we nest the `p` selector? Not tested, but something like:

```scss
  .govuk-checkboxes__item {
    label {
      @extend .govuk-\!-font-weight-bold;
    }

    .guidance {
       display: block;
       padding-right: govuk-spacing(3);
       padding-left: govuk-spacing(3);
    }
    
    .hint {
      display: block;
      padding-right: govuk-spacing(3);
      padding-left: govuk-spacing(3);

      > p {
        @extend .govuk-hint;
      }
    }
```

Reply:
We don't need the divs - but one could argue it makes them more specific?

Happy to change it to the suggestion.

Reply:
I really like how the nesting in Sass reflects the HTML structure